.therapist.yml

```diff
@@ -1,22 +1,23 @@
 actions:
   eslint:
     description: eslint
-    run: yarn eslint {files}
-    fix: yarn eslint --fix {files}
+    run: ./node_modules/.bin/eslint --color --max-warnings 0 {files}
```

Comment:
Why this set of changes?

Reply:
- the switch from yarn to the node_modules path was because yarn adds all this extra noise to the therapist output. 
- `--color` is just to force color output which improves legibility (imo) when there's a long list of errors.
- `--max-warnings` could go in another PR but since I was making changes I figured I'd sneak this in. It addresses something I found in another branch where I was converting stuff to typescript where there were a bunch of ESLint warnings like `Missing return type on function` and `Unexpected any. Specify a different type`.